### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Oshratn**
## Overview
Rewrite to highlight how far Kubescape has come beyond being just a misconfiguration scanner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,9 +18,11 @@
   <img alt="Kubescape logo" align="right" src="https://raw.githubusercontent.com/cncf/artwork/master/projects/kubescape/stacked/color/kubescape-stacked-color.svg" width="150">
 </picture>
 
-_An open-source Kubernetes security platform for your IDE, CI/CD pipelines, and clusters_
+_An open-source Kubernetes security platform for your IDE, CI/CD pipelines, and clusters that seperates out security signal from scanner noise_
```
Comment (slashben) [inline README.md:21]: I'd reverse the order: Clusters, CI/CD pipelines and IDE

File: README.md [hunk 2]
```diff
@@ -18,9 +18,11 @@
...
 
-Kubescape is an open-source Kubernetes security platform. It includes risk analysis, security compliance, and misconfiguration scanning. Targeted at the DevSecOps practitioner or platform engineer, it offers an easy-to-use CLI interface, flexible output formats, and automated scanning capabilities. It saves Kubernetes users and admins precious time, effort, and resources.
+Kubescape is an open-source Kubernetes security platform, built for use in your day-to-day workflow, by fitting into your IDE, CI/CD pipelines and clusters. It serves as a one-stop-shop for Kuberenetes security. You can run scans via the CLI, or add the Kubescape Helm chart, which gives an in-depth view of what is going on in the cluster.
```
Comment (slashben) [inline README.md:23]: > which gives an in-depth view of what is going on in the cluster
I'd rather rephrase this around monitoring (that the CLI cannot do)

**Review by rotemamsa — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -18,9 +18,11 @@
...
+
+Kubescape performs misconfiguration and vulnerability scanning and provides risk analysis and security compliance indicators. All results are presented in context and users get many cues on what to do based on scan results. Targeted at the DevSecOps practitioner or platform engineer, it offers an easy-to-use CLI interface, flexible output formats, and automated scanning capabilities. It saves Kubernetes users and admins precious time, effort, and resources in the race to keep continuously secure and compliant.
```
Comment (slashben) [inline README.md:25]: I would not go into the "ICP" language by saying "DevSecOps or platform engineer". Open source project are targeting everyone interested 😄

**Review by slashben — CHANGES_REQUESTED** [review]
see my comments

**Comment by craigbox** [comment]
Whatever you do here, also do to [the index of kubescape/kubescape.io](https://github.com/kubescape/kubescape.io/blob/main/docs/index.md).

File: README.md (hunk 2 above)
Comment (Oshratn) [reply README.md]: Not sure what you meant here. 
The reason I phrased it the way I did is that I wanted to highlight that agentess scans are available and to get more info you install the operator.

File: README.md (hunk 3 above)
Comment (Oshratn) [reply README.md]: Removed the ICP, as requested.

File: README.md (hunk 2 above)
Comment (slashben) [reply README.md]: Changing 
> You can run scans via the CLI, or add the Kubescape Helm chart, which gives an in-depth view of what is going on in the cluster
to something like
"You can run scans via the CLI, or add the Kubescape Helm chart for continuous monitoring of the cluster"

**Comment by Oshratn** [comment]
>Whatever you do here, also do to [the index of kubescape/kubescape.io](https://github.com/kubescape/kubescape.io/blob/main/docs/index.md).

Once fully approved I will create a PR ofr Kubescape.io

**Review by rotemamsa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (slashben) [reply README.md]: @Oshratn this was not addressed

**Pull request closed without merging**